travel_platform/smart_ordering.py

```diff
@@ -24,14 +24,13 @@ def smart_ordering(data_dict, filter_by, order_by='ASC', limit=100):
     '''
     if order_by == 'DESC':
         sorted_data = sorted(data_dict,
-                             key=lambda note: note.get(filter_by, 'z'),
+                             key=lambda note: (filter_by not in note, note.get(filter_by)),
                              reverse=True)
     else:
         sorted_data = sorted(data_dict,
-                             key=lambda note: note.get(filter_by, 'z'))
+                             key=lambda note: (filter_by not in note, note.get(filter_by)))
```

Comment:
BTW, I can't get one thing about this sorting condition. When I run your function this way -
 smart_ordering(data_dict=data, filter_by='age', order_by='DESC', limit=5) it returns 5 dicts where there is no age at all. Is it ok?I expected to see people sorted by age.